frappe/public/js/frappe/views/reports/report_view.js

```diff
@@ -789,7 +789,10 @@ frappe.views.ReportView = class ReportView extends frappe.views.ListView {
 			} else {
 				this.fields.splice(col_index, 0, field);
 			}
-			frappe.show_alert(__('Also adding the dependent currency field {0}', [field[0].bold()]));
+			const field_label = frappe.meta.get_label(doctype, field[0]);
```

Comment:
Labels are not necessarily unique
Either we should use only fieldnames or fieldnames + label
`Label (fieldname)`

Reply:
The user never gets to see the fieldname, only the label (lists, filters, reports, forms, ...). I don't know why this should be different here.

Reply:
Because over here when he sees a message and there are two fields with the same label the user should understand which field are we talking about.
informs, there are fields related to each other placed close to each other and under sections.